UserInterface/ToolbarItemDemos/ToolbarItemDemos/ToolbarItemDemos/ToolbarItemXaml.xaml

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:d="http://xamarin.com/schemas/2014/forms/design"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             mc:Ignorable="d"
+             x:Class="ToolbarItemDemos.ToolbarItemXaml"
+             Padding="10"
+             Title="Xaml ToolbarItem Demo">
+<ContentPage.ToolbarItems>
```

Comment:
Indentation of this block. Does VS not format on save for XAML? VSMac does.

Similarly, each attribute/property on a new line (VSMac format on save does this).

Reply:
It does not. And when I auto-format it often misaligns properties. Our convention is to align them with the property on the above line. I believe VS for PC aligns them versus traditional code indentation to the nearest "tab".